statsmodels/stats/tests/test_contingency_tables.py

```diff
@@ -525,3 +528,442 @@ def __init__(self):
+    spmi_bonferroni_r = pd.DataFrame.from_csv(fpath)
+
+    table_p_value_r = spmi_bonferroni_r["p.value.bon"]
+    cell_p_values_r = spmi_bonferroni_r.iloc[:, 1:]
+
+    assert_allclose(cellwise_p_bonferroni_python, cell_p_values_r)
+    assert_allclose(table_p_value_r, table_p_value_bonferroni_corrected)
+
+
+def test_SPMI_false_using_rao_scott_2():
+    rows_factor = ctab.Factor.from_array(presidential_data.iloc[:, 6:11], presidential_data.columns[6:11],
+                              "believe_true", orientation="wide")
+    columns_factor = ctab.Factor.from_array(presidential_data.iloc[:, 11:], presidential_data.columns[11:],
+                                 "why_uncertain", orientation="wide")
+    multiple_response_table = ctab.MultipleResponseContingencyTable([rows_factor, ], [columns_factor])
+    rao_scott_2_test = multiple_response_table._test_for_single_pairwise_mutual_independence_using_rao_scott_2
+    table_p_value = rao_scott_2_test(rows_factor, columns_factor)
+    fpath = os.path.join(results_dirpath, "spmi_r_rao_scott.csv")
+    r_result = pd.DataFrame.from_csv(fpath)
+    table_p_value_r = r_result["p.value.rs2"]
+    assert_allclose(table_p_value_r, table_p_value)
+
+
+def build_random_single_select(n=10000, choices=None):
+    if choices:
+        k = len(choices)
+    else:
+        k = 3
+        choices = ["sedan", "truck", "motorcycle"]
+    car_type = np.random.randint(k, size=(n)) + 1
+    base_pop = pd.DataFrame(car_type).reset_index()
+    base_pop.columns = ['person', 'choice']
+    base_pop['_response'] = 1
+    dataframe = pd.pivot_table(base_pop,
+                               values='_response',
+                               fill_value=0,
+                               index='person',
+                               columns='choice',
+                               aggfunc=np.sum,
+                               margins=False)
+    car_choice = dataframe.copy()
+    car_choice.columns = choices
+    return car_choice
+
+
+def test_multiple_mutual_independence_true():
+    np.random.seed(100)
+    food_choices = pd.DataFrame(np.random.randint(2, size=(10000, 5)),
+                                columns=["eggs", "cheese", "candy", "sushi", "none"])
+    car_choice = build_random_single_select()
+    srcv = ctab.Factor.from_array(car_choice, car_choice.columns, "car_choice", orientation="wide")
+    mrcv = ctab.Factor.from_array(food_choices, food_choices.columns, "food_choices", orientation="wide")
+    multiple_response_table = ctab.MultipleResponseContingencyTable([srcv, ], [mrcv, ])
+    rao_scott_2_test = multiple_response_table._test_for_marginal_mutual_independence_using_rao_scott_2
+    rao_p_value = rao_scott_2_test(srcv, mrcv)
+    assert rao_p_value >= 0.05
+    bonferroni_test = multiple_response_table._test_for_marginal_mutual_independence_using_bonferroni_correction
+    bonferroni_table_p_value, bonferroni_cell_p_values = bonferroni_test(srcv, mrcv)
+    assert bonferroni_table_p_value >= 0.05
```

Comment:
no plain assert in unit tests, either use generic np.testing.assert_ or a specific assert_xxx, e.g. assert_array_less.